src/main/kotlin/app/revanced/patches/reddit/ad/general/patch/GeneralAdsPatch.kt

```diff
@@ -0,0 +1,55 @@
+package app.revanced.patches.reddit.ad.general.patch
+
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.data.impl.BytecodeData
+import app.revanced.patcher.extensions.replaceInstruction
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patcher.patch.impl.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patches.reddit.ad.general.annotations.GeneralAdsCompatibility
+import org.jf.dexlib2.Opcode
+import org.jf.dexlib2.builder.instruction.BuilderInstruction21c
+import org.jf.dexlib2.iface.instruction.ReferenceInstruction
+import org.jf.dexlib2.iface.reference.StringReference
+import org.jf.dexlib2.immutable.reference.ImmutableStringReference
+
+@Patch
+@Name("general-ads")
```

Comment:
Conflicts with another patch:
```suggestion
@Name("general-reddit-ads")
```